society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -35,7 +35,14 @@
       <term name="editor" form="verb-short">ed.</term>
       <term name="translator" form="verb-short">trans.</term>
       <term name="translator" form="short">trans.</term>
-      <term name="editortranslator" form="verb">edited and translated by</term>
+      <term name="editortranslator" form="verb-short">
+        <single>ed. and trans.</single>
+        <multiple>ed. and trans.</multiple>
+      </term>
+      <term name="editortranslator" form="verb">
+        <single>edited and translated by</single>
+        <multiple>edited and translated by</multiple>
+      </term>
```

Comment:
@fbennett, is there a logic why this should be necessary?

The locales-en-US.xml locale file only defines a single form for these terms. See e.g. https://github.com/citation-style-language/locales/blob/2fddb0b00c15c04effe6fb46a3e74175f10896b7/locales-en-US.xml#L321.

Reply:
@fbennett ping -- I'm happy to take this, but I'd also like to understand why it's necessary.

Reply:
The `editortranslator` trick works by calling a locale term of the same form with that name, when the call to `cs:names` includes exactly `editor` and `translator`. The `editor-translator` macro in SBL applies `verb-short`, which exists in `locales-en-US.xml`, so without anything more, it should work.

I don't remember the precise overloading hierarchy, but loading a term of a particular form will clobber its subordinates. Where SBL loads an `editortranslator` term in its `locale` section, I think that wipes out the `verb-short` version on which collapsing actually depends. Redefining it explicitly enabled the collapse.

If you want to trim this down, it looks like the `verb` form isn't used in the style, so I think that could be removed. Also, since the `single` and `multiple` forms are identical, defining the term value as a single static element should work fine. Would you like me to make those changes?